eng/Versions.props

```diff
@@ -9,27 +9,34 @@
     <UsingToolSymbolUploader>true</UsingToolSymbolUploader>
     <UsingToolVSSDK>true</UsingToolVSSDK>
   </PropertyGroup>
+
```

Comment:
Nit: The Arcade update bot is going to remove all empty lines the next time it comes through.